_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+html {
+    height: 102%;
+}
+
+body {
+    font-size: $font-size;
+    min-width: $fg-min-width;
```

Comment:
This is not needed now - it pertains to a grid we don't have and uses Bourbon's flex-grid (http://bourbon.io/docs/#flex-grid) which we won't be using when we get there.